apps/chat/src/components/Chat/ChatSettingsEmpty.tsx

```diff
@@ -10,7 +10,7 @@ interface Props {
   isModels: boolean;
   conversation: Conversation;
   prompts: Prompt[];
-  defaultModelId: string;
+  defaultModelId: string | undefined;
```

Comment:
Why can we have no default model?

Reply:
We don't have any default model constant anymore, so if default model not set in ENV variables we will not have default model

Reply:
@mikitabut , what will be selected for new conversation (after clear recent model ids and clear all)?

Reply:
Actually you are right - we still have fallback model
Fixed it